pkgs/tools/graphics/shotgun/default.nix

```
@@ -0,0 +1,27 @@
+{ pkgs, stdenv, fetchFromGitHub, rustPlatform, ... }:
```

Do not rely on `pkgs` and pass all dependencies explicitly:

```suggestion
{ lib, fetchFromGitHub, rustPlatform, pkg-config, libXrandr, libX11 }:
```